src/test/java/com/eainde/assignment/EmployeeServiceTest.java

```diff
@@ -108,21 +108,21 @@ void getEmployeeById() {
 
     @Test
     @DisplayName("Checking if the method is returning highest salary employee")
-    void getHighestSalaryEmployee() {
+    void getHighestSalaryEmployee() throws IOException {
         Employee employee = getInstance().getHighestSalaryEmployee();
         Assertions.assertEquals(employee.getId(), 4);
         Assertions.assertEquals(employee.getName(), "William");
     }
 
     @Test
     @DisplayName("Checking if the method is returning average salary")
-    void getAverageSalary() {
+    void getAverageSalary() throws IOException {
         Assertions.assertEquals(getInstance().getAverageSalary(), 46800.09);
     }
 
     @Test
     @DisplayName("Checking if the method is returning all the employees by city")
-    void getEmployeesByCity() {
+    void getEmployeesByCity() throws IOException {
```

Comment:
This exception should be handled in the main method so that you don't have to declare int the test.